connectors/sources/mysql.py

```diff
@@ -454,11 +444,12 @@ async def close(self):
         self._sleeps.cancel()
 
     async def validate_config(self):
-        """Validates whether user input is empty or not for configuration fields and validate type for port.
-        Also validate, if the configured database and the configured tables are present and accessible by the configured user.
+        """Validates that user input is not empty and adheres to the specified constraints.
+        Also validate, if the configured database and the configured tables are present and accessible using the configured user.
 
         Raises:
-            Exception: Configured keys can't be empty
+            ConfigurableFieldValueError: The database or the tables do not exist or aren't accessible/a field contains an empty or wrong value
```

Comment:
Absolute nit:
 (the `/` didn't read as separating sentences)

```suggestion
            ConfigurableFieldValueError: The database or the tables do not exist or aren't accessible, or a field contains an empty or wrong value
```

Reply:
Looks better!